### PR TITLE
Add support for Razer Mamba Wireless

### LINKED ISSUE #747: Support for Razer Mamba Wireless (1532:0072/0073)
### Device Info

* Name: Razer Mamba Wireless
* Product Number: RZ01-02710100
* Model Number: RC30-027101

### Device Information

Please post the output of
    lsusb -d 1532: -v 

[lsusbcap.txt](https://github.com/openrazer/openrazer/files/2710489/lsusbcap.txt)

Please post the output of
    ls -lah /dev/input/by-id/

[lscap.txt](https://github.com/openrazer/openrazer/files/2710487/lscap.txt)

### Synapse Screenshots

calibration
customize
lighting
performance
power

### PR DESCRIPTION
**Pull request by crian**
Fixes https://github.com/openrazer/openrazer/issues/747

### CODE REVIEW

File: daemon/openrazer_daemon/hardware/mouse.py [hunk 1]
```diff
@@ -1017,6 +1017,112 @@ class RazerMamba2012Wired(__RazerDeviceSpecialBrightnessSuspend):
...
+        """
+        logo_brightness = self.suspend_args.get('brightness', (100, 100))[0]
+        scroll_brightness = self.suspend_args.get('brightness', (100, 100))[1]
+        self.disable_notify = True
+        _da_set_logo_brightness(self, logo_brightness)
+        _da_set_scroll_brightness(self, scroll_brightness)
+        self.disable_notify = False
+
+
+class RazerMambaWirelessWired(__RazerDeviceSpecialBrightnessSuspend):
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/mouse.py:1020]: You can make this class inherit the other class and remove all the common stuff (which seems to be everything except `USB_PID = 0x0073` and `EVENT_FILE_REGEX`

**Review by z3ntu — CHANGES_REQUESTED** [review]
LGTM except for a small comment. Thanks for the PR!

File: daemon/openrazer_daemon/hardware/mouse.py (hunk 1 above)
Comment (crian) [reply daemon/openrazer_daemon/hardware/mouse.py:1020]: First, sorry for the late respond.
Pushed the requested changes.

**Review by z3ntu — APPROVED** [review]
LGTM, thank you very much for the PR!

**Pull request merged**